config/develop/example-app-1-study-1.yaml

```diff
@@ -0,0 +1,34 @@
+template_path: study-pipeline-infra.j2
+stack_name: example-app-1-study-1
+parameters:
+  BranchOrTagName: ETL-93/add-versioning
+  AppName: example-app-1
+  StudyName: study-1
+  TemplateBucketName: {{ stack_group_config.artifact_bucket_name }}
+  SourceBucketName: !stack_output_external bridge-downstream-dev-source-bucket::BucketName
+  JsonBucketName: !stack_output_external bridge-downstream-dev-intermediate-bucket::BucketName
+  ParquetBucketName: !stack_output_external bridge-downstream-dev-parquet-bucket::BucketName
+  RoleArn: !stack_output_external glue-job-role::RoleArn
+  ClassifierName: !stack_output_external array-of-records-classifier::ClassifierName
+  SynapseAuthSsmParameterName: synapse-bridgedownstream-auth
+  UniqueId: !rcmd git rev-parse HEAD
+
+stack_tags:
+  {{ stack_group_config.default_stack_tags }}
+
+sceptre_user_data:
+  app_version: 'version 1.0.2, build 57'
+  dataset_version_mapping: !file src/glue/resources/dataset_mapping.json
+  dataset_crawler_assignments:
+    standard:
+      - taskresult
```

Comment:
in person code review: @philerooski suggests making crawler assignments aware of version